R/solvebio.R

```diff
@@ -40,9 +40,15 @@ login <- function(api_key, api_host, env = solvebio:::.solveEnv) {
     }
 
     if(nchar(env$token) == 0L) {
-        stop("No API key found. Please set the 'SOLVEBIO_API_KEY' environment variable, or specify your key as the 'api_key' parameter of this function. Your API key can be found on the Account page of the SolveBio website: https://my.solvebio.com/settings/security")
+        # No API key, look for access token
+        .solveEnv$token <- Sys.getenv('SOLVEBIO_ACCESS_TOKEN', unset='')
```

Comment:
Can you add the check for API key above this? Check api key, then token if key is missing. Basically repeating what's at the top of the file.